src/displayPokes.js

```diff
@@ -1,20 +1,62 @@
+import popUp from './pop_up.js';
+
 const displayPokemon = (pokemon) => {
-  const cardSection = document.getElementById('card-section');
-  const pokemonTypes = pokemon.types;
-  const pokemonImage = pokemon.sprites.other;
-
-  let displayHTML = `<li class="card">
-  <img src="${pokemonImage['official-artwork'].front_default}" alt="Pokemon">
-  <h1>${pokemon.name}</h1>
-  <ul>`;
-  for (let i = 0; i < pokemonTypes.length; i += 1) {
-    displayHTML += `<li>${pokemonTypes[i].type.name}</li>`;
-  }
-  displayHTML += `</ul>
-  <button type="submit">Comment</button>
-</li>`;
-
-  cardSection.innerHTML += displayHTML;
-};
+  pokemon.forEach((pkm) => {
+    fetch(pkm.url)
+      .then((response) => response.json())
+      .then((pkmData) => {
+        const cardSection = document.getElementById('card-section');
+        const pokemonTypes = pkmData.types;
+        const pokemonImage = pkmData.sprites.other;
+
+        const outerLi = document.createElement('li');
+        outerLi.classList.add('card');
+
+        let displayHTML = `
+        <img src="${pokemonImage['official-artwork'].front_default}" alt="Pokemon">
+        <h1>${pkmData.name}</h1>
+        <ul>`;
+        for (let i = 0; i < pokemonTypes.length; i += 1) {
+          displayHTML += `<li>${pokemonTypes[i].type.name}</li>`;
+        }
+        displayHTML += '</ul>';
+
+        outerLi.innerHTML = displayHTML;
+
+        const btn = document.createElement('button');
+        btn.setAttribute('type', 'button');
+        btn.classList.add(pkmData.name, 'btn');
+        btn.innerHTML = 'Comment';
+        btn.addEventListener('click', () => {
+          popUp(pkmData);
+        });
+
+        outerLi.appendChild(btn);
 
+        cardSection.appendChild(outerLi);
+      });
+  });
+  //   pokemon.forEach((pkm) => {
```

Comment:
Kindly remove the commented lines of code. This would make the code look cleaner as those lines of code are currently not in use.

Reply:
Requested change completed.